monkey/monkey_island/cc/services/initialize.py

```diff
@@ -101,8 +101,7 @@ def _register_repositories(container: DIContainer, data_dir: Path):
     )
     container.register_instance(IUserRepository, container.resolve(JSONFileUserRepository))
 
-    # TODO: Replace with MongoEventRepository
-    container.register_instance(IAgentEventRepository, StubbedEventRepository())
+    container.register_instance(IAgentEventRepository, container.resolve(MongoEventRepository))
```

Comment:
`MongoEventRepository` uses `EventSerializerRegistry` which is registered later in the flow. @cakekoa fixed it in `2180-encrypt-event-data`.